pandas/core/window/ewm.py

```diff
@@ -502,7 +502,19 @@ def aggregate(self, func, *args, **kwargs):
         create_section_header("See Also"),
         template_see_also,
         create_section_header("Notes"),
-        numba_notes.replace("\n", "", 1),
+        numba_notes,
+        create_section_header("Examples"),
+        dedent(
+            """\
+        >>> ser = pd.Series([1, 2, 3, 4])
+        >>> ser.ewm(alpha=.2).mean()
```

Comment:
nice! I don't know how hard it is, but for a simple example like this one, would it be feasible to show how the result is calculated? like, to match it with just simple numpy operations?